src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,126 @@
 /* eslint-disable */
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { loadedTodos } from '../../features/todos';
+import classNames from 'classnames';
+import { selectedTodo, setCurrentTodo } from '../../features/currentTodo';
+import { Todo } from '../../types/Todo';
+import { selectQuery, selectStatus } from '../../features/filter';
+
+export enum FILTER {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
+}
```

Comment:
The `FILTER` enum is correctly defined here. Ensure that this definition is consistently used across the application, especially in the `TodoFilter` component, to avoid any mismatches or runtime errors.